src/swsssdk/configdb.py

```diff
@@ -142,18 +142,21 @@ def __raw_to_typed(self, raw_data):
         return typed_data
 
     def __typed_to_raw(self, typed_data):
+        raw_data = dict(); dfields = dict()
         if typed_data == None:
-            return None
+            return raw_data, dfields
         elif typed_data == {}:
-            return { "NULL": "NULL" }
-        raw_data = {}
+            return { "NULL": "NULL" }, dfields
+
         for key in typed_data:
             value = typed_data[key]
             if type(value) is list:
                 raw_data[key+'@'] = ','.join(value)
+            elif value is None:
+                dfields[key] = None
             else:
                 raw_data[key] = str(value)
-        return raw_data
```

Comment:
With this change, the raw_data could potentially become empty. If so, guess, you would need to return as {"NULL": "NULL"}  -- same as in line 149